subsys/bluetooth/rpc/client/bt_rpc_crypto_client.c

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <sys/types.h>
+
+#include "nrf_rpc_cbor.h"
+
+#include "bluetooth/crypto.h"
+
+#include "bt_rpc_common.h"
+#include "serialize.h"
+
+
+struct bt_rand_rpc_res {
+	int result;
+	size_t len;
+	uint8_t *buf;
+};
+
+static void bt_rand_rpc_rsp(CborValue *value, void *handler_data)
+{
+	struct bt_rand_rpc_res *res =
+		(struct bt_rand_rpc_res *)handler_data;
+
+	res->result = ser_decode_int(value);
+	ser_decode_buffer(value, res->buf, sizeof(uint8_t) * res->len);
+}
+
+int bt_rand(void *buf, size_t len)
+{
+	struct nrf_rpc_cbor_ctx ctx;
+	struct bt_rand_rpc_res result;
+	size_t scratchpad_size = 0;
+	size_t buffer_size_max = 10;
+
+	scratchpad_size += SCRATCHPAD_ALIGN(sizeof(uint8_t) * len);
+
+	NRF_RPC_CBOR_ALLOC(ctx, buffer_size_max);
+	ser_encode_uint(&ctx.encoder, scratchpad_size);
+	ser_encode_uint(&ctx.encoder, len);
+
+	result.len = len;
+	result.buf = buf;
+
+	nrf_rpc_cbor_cmd_no_err(&bt_rpc_grp, BT_RAND_RPC_CMD,
+		&ctx, bt_rand_rpc_rsp, &result);
+
+	return result.result;
+}
+
+int bt_encrypt_le(const uint8_t key[16], const uint8_t plaintext[16],
+		  uint8_t enc_data[16])
+{
+	struct nrf_rpc_cbor_ctx ctx;
+	size_t key_size;
+	size_t plaintext_size;
+	size_t enc_data_size;
+	int result;
+	size_t scratchpad_size = 0;
+	size_t buffer_size_max = 20;
+
+	key_size = sizeof(uint8_t) * 16;
+	buffer_size_max += key_size;
+	plaintext_size = sizeof(uint8_t) * 16;
+	buffer_size_max += plaintext_size;
+	enc_data_size = sizeof(uint8_t) * 16;
+	buffer_size_max += enc_data_size;
+
+	scratchpad_size += SCRATCHPAD_ALIGN(key_size);
+	scratchpad_size += SCRATCHPAD_ALIGN(plaintext_size);
+	scratchpad_size += SCRATCHPAD_ALIGN(enc_data_size);
+
+	NRF_RPC_CBOR_ALLOC(ctx, buffer_size_max);
+	ser_encode_uint(&ctx.encoder, scratchpad_size);
+
+	ser_encode_buffer(&ctx.encoder, key, key_size);
+	ser_encode_buffer(&ctx.encoder, plaintext, plaintext_size);
+	ser_encode_buffer(&ctx.encoder, enc_data, enc_data_size);
+
+	nrf_rpc_cbor_cmd_no_err(&bt_rpc_grp, BT_ENCRYPT_LE_RPC_CMD,
+		&ctx, ser_rsp_decode_i32, &result);
+
+	return result;
+}
+
+
+
```

Comment:
Excessive blank lines. Could it be a generator artifact?

Reply:
It could be. I went through files and removed all unnecessary blank lines